flipperdevices/flipperzero-firmware
# [FL-3583] Fix multiline aligned text going out of bounds (again)

# What's new

- Repeat of #3038 that doesn't corrupt the string

# Verification 

- See #3038
- Check that the final string isn't missing any characters

# Checklist (For Reviewer)
